.github/workflows/linter.yml

```diff
@@ -61,19 +61,5 @@ jobs:
       ################################
       # Run Linter against code base #
       ################################
-      - name: Lint Code Base
-        uses: github/super-linter@v3
-        env:
-          VALIDATE_ALL_CODEBASE: false # Validate only changed code
-          DEFAULT_BRANCH: main
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          LINTER_RULES_PATH: /
-          # JAVASCRIPT_DEFAULT_STYLE: prettier
-          # JAVASCRIPT_ES_CONFIG_FILE: .eslintrc.json
-          TYPESCRIPT_ES_CONFIG_FILE: .eslintrc
-          # EDITORCONFIG_FILE_NAME: .editorconfig
-          VALIDATE_TYPESCRIPT_ES: true
-          #VALIDATE_EDITORCONFIG: true
-          #VALIDATE_HTML: true
-          #VALIDATE_JSON: true
-          #VALIDATE_MARKDOWN: true
+      - name: Run ESLint
```

Comment:
I think it would be nice to keep the github linter and just put `VALIDATE_TSX: true` to enable support for `.tsx` files using the eslint config we have.
Also adding `JAVASCRIPT_DEFAULT_STYLE: prettier` should solve the prettier problem I think.

Reply:
What integration does the GitHub linter provide that we don't have with eslint? According to their [README](https://github.com/github/super-linter/blob/master/README.md), it's just a collection of linters for different languages that prints output to the console and adds a status check to PRs. Running eslint manually does the same.

I tried adding these options. Looks like `.tsx` files are linted correctly now (at least partially, idk why it's run twice with different results: [log1](https://github.com/ProjectMilou/frontend/pull/85/checks?check_run_id=2184045301), [log2](https://github.com/ProjectMilou/frontend/runs/2184045581?check_suite_focus=true)), but it's extremely slow and takes about 10 seconds per file (more than 7 minutes if all files are checked, compared to 20 seconds total for eslint). Even if only changed files are linted, this is unacceptable imo.

`JAVASCRIPT_DEFAULT_STYLE: prettier` does not seem to have any effect on the output. No prettier errors are reported, but there should be some in the most recent run ("prettier check" is still the yarn script, I didn't remove it).

Maybe these problems are fixable, but in the end we can never be sure that this linter does exactly what's specified in our eslint config, so I still think we should just run eslint and prettier directly.